lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -28,8 +27,15 @@
     "auditMode": false,
     "gatherMode": "lighthouse-core/test/results/artifacts",
     "disableStorageReset": false,
-    "emulatedFormFactor": "mobile",
-    "internalDisableDeviceScreenEmulation": false,
+    "formFactor": "mobile",
+    "emulatedUserAgent": "Mozilla/5.0 (Linux; Android 7.0; Moto G (4)) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4143.7 Mobile Safari/537.36 Chrome-Lighthouse",
```

Comment:
I guess the UA string count in the LHR is getting a bit ridiculous but not really a clean way to cut back :)